semcore/radio/src/style/radio.shadow.css

```diff
@@ -118,3 +118,10 @@ SControl:checked ~ SValue[theme]::before {
 SValue[state='normal'][theme]::before {
   border-color: var(--theme);
 }
+
+SInvalidPattern {
+  height: calc(100% - 2px);
+  margin: 1px 0 1px 1px;
+  border-bottom-left-radius: 12px;
```

Comment:
Probably you should use border radiuses from design tokens

Reply:
Also, does it work well with `l` radio size?

Reply:
Sure! It was me who experimented with styles and forgot to add them. I'll fix!

Reply:
Updated!